### PR TITLE
feat: index carbon react

### LINKED ISSUE #451: Index Carbon React
- [x] Inherit styles library (matching ids)
- [x] Links to docs, demos
- [x] Status
- [x] Reintroduce carbonComponentsGroup (services/web-app/data/libraries.js)
- [x] Hooks
- [x] Add thumbnails

image

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #451 

Testing / reviewing
Run the app locally you should see `carbon react components` listed as a library and the components should render (61 total) with the filters as shown below:

163252174-91bc06a9-497d-4080-a238-12fafe6597e7

### CODE REVIEW

**Review by francinelucca — COMMENTED** [review]
Looks really good to me so far, can see 63 components :tada 
image


QQ: is stack supposed to start lowecase? 
image

**Comment by andreancardona** [comment]
> Looks really good to me so far, can see 63 components :tada image
> 
> QQ: is stack supposed to start lowecase? image

Good catch! Just updated 🔥 `stack -> Stack`

File: services/web-app/data/libraries.js [hunk 1]
```diff
@@ -93,12 +91,12 @@ const prodLibraries = {
...
-  //   host: 'github.ibm.com',
-  //   org: 'matt-rosno',
+  //   host: 'github.com',
+  //   org: 'carbon-design-system',
   //   repo: 'carbon',
-  //   path: '/packages/carbon-react',
+  //   path: '/packages/react',
   //   sponsor: 'carbon',
-  //   group: carbonComponentsGroup
+  //   group: carbonComponentsGroup,
```
Comment (andreancardona) [inline services/web-app/data/libraries.js:99]:
```suggestion
  //   group: carbonComponentsGroup
```

**Review by francinelucca — COMMENTED** [review]
Looks good! confirmed only missing fluidform and toggletip thumbs in carbon react, punchouts look goof, library details loog good 👍 .

This should be good to merge once we get https://github.com/carbon-design-system/carbon/pull/11208 in 💃

**Comment by andreancardona** [comment]
@alisonjoseph toggletip is updated :)

**Review by alisonjoseph — DISMISSED** [review]
LGTM!

**Review by alisonjoseph — APPROVED** [review]

**Review by francinelucca — APPROVED** [review]
🎉

**Pull request merged**